autoscrub/scripts/cli.py

```diff
@@ -38,7 +40,25 @@ def check_ffmpeg():
     except (subprocess.CalledProcessError, OSError):
         click.echo("[autoscub:error] Could not find ffprobe executable. Check that ffprobe is in the local folder or your system PATH and that you can run 'ffprobe -L' from the command line.")
         raise click.Abort()
-        
+
+    # check ffmpeg version

```

Comment:
Is this not the same code as `autoscrub.version_check.check_ffmpeg_version`?

Reply:
More or less, save for the `print` statements being replaced with `click.echo` which wouldn't make sense in `autoscrub.version_check.check_ffmpeg_version` (which could be omitted).